### ISSUE TITLE
PV-RCNN++ trainable on custom dataset but with AP of 0 after 6000 Epochs

### ISSUE THREAD

**Issue by fjahncke** [open-mmlab/OpenPCDet#1280]
I would like to evaluate the performance of PV-RCNN++ on my custom dataset. I tried converting the yaml file from the Waymo dataset unsucessfully and running the train.py on it throws the Error: `'EasyDict' object has no attribute 'FEATURE_MAP_STRIDE'`. I then changed the config file to the solution from this issue: [https://github.com/open-mmlab/OpenPCDet/issues/829#issuecomment-1065878449](url), so changing the Dense Head, and can train my network with a decreasing loss function for 6000 Epochs. Sadly after the entire training the AP remains 0 and I cant detect anything
Is there any way to get PV-RCNN++ running on a custom dataset and are any other changes to the config file necessary? I can succesfully train and evaluate PV-RCNN, SECOND, and CenterPoint on the same custom dataset

**Comment by fjahncke** [comment]
Possibly discovered my own solution and just documenting it of other people run into the same problem, changing from PV-RCNN++ to PV-RCNN++ (ResNet) works even with CenterHead as Dense Head. I would therefore assume its a problem with the 3D Backbone. I didnt change anything in the config file of  PV-RCNN++ (ResNet) except for the evaluation metric and the obvious changes at the top
